src/app/modules/group/pages/group-access/group-access.component.ts

```diff
@@ -13,11 +16,11 @@ import { mapToFetchState } from '../../../../shared/operators/state';
 export class GroupAccessComponent implements OnChanges, OnDestroy {
   @Input() group?: Group;
 
-  private readonly rootActivityId$ = new ReplaySubject<string | null>(1);
+  private readonly group$ = new ReplaySubject<Group>(1);
 
   private refresh$ = new Subject<void>();
```

Comment:
Since there are 2 refresh now, we should probably rename this one as well

Reply:
refresh$ is for rootActivityState$, which is another section. I have suggestion for that in optional comment of this task